### ISSUE TITLE
CONTRIBUTING.md - Design standards 

### ISSUE THREAD

**Issue by eosrei** [EmojiTwo/emojitwo#67]
The current `CONTRIBUTING.md` describes color schemes, but doesn't cover overall design standards. Should artists restrict themselves to the original EmojiOne style? I believe so. Either way, this should be clearly stated.

**Comment by Crissov** [comment]
I agree, but except for colors, stroke widths and orientation this is beyond me to specify. I'm planning to develop a reasonable system for colors used in symbols, for instance. 

Emojione also has some emojis deviate significantly from its usual design, e.g. :articulated_lorry: or :racing_car: vs. :truck: :car: :taxi: :police_car: :fire_engine:. These should be recorded as issues. (Some of them could be imported from <http://github.com/emojione/emojione-assets/issues/>.)
